lambdas/models/bulk_upload_report_output.py

```diff
@@ -70,6 +71,7 @@ def populate_report(self):
                 self.process_failed_report_item(item)
 
         self.set_unique_failures()
+        self.set_total_ingested()
```

Comment:
Is there no way we can just do the count of total ingested in the for loop above? Just store each `item.nhs_number` in the ReportBase total_ingested as a set() rather than an int. This way it will only count unique nhs numbers. Then in the summary, similar to the other calculations we do `total_ingested=summary_report.get_total_ingested_count()`?

Reply:
`self.total_ingested.add(item.nhs_number)`